cookbooks/zarr-stac-report/data-consumers/index.qmd

```diff
@@ -0,0 +1,98 @@
+---
+title: "Zarr + STAC for Data Consumers"
+subtitle: "Accessing data cubes in Zarr + STAC"
+---
+
+This section is targeted at people who are trying to access data cubes in Python. It addresses questions like:
```

Comment:
Does this only apply to Python?

Reply:
Maybe good to say that Python is the most elaborated but the concepts apply to other Zarr access tools ...

Reply:
Good point. I am quick to see the world as Python-only.